src/components/interaction-designer/toolbar/ErrorNotifications.vue

```diff
@@ -111,26 +111,19 @@ export class ErrorNotifications extends mixins(Routes, Lang) {
     return pickBy(this.validationStatuses, (value: IValidationStatus, key) => key.includes('block/') && blocksMap[key])
   }
 
-  get resourceValidationStatusesForActiveFlow(): Dictionary<IValidationStatus> {
-    return pickBy(this.validationStatuses, (status, key) => {
-      const resourceUuid = replace(key, 'resource/', '')
-      return this.resourceUuidsOnActiveFlow.includes(resourceUuid) && !get(this.validationStatuses, `resource/${resourceUuid}`)?.isValid
-    })
-  }
-
   hasBlockValidationErrors(uuid: string): boolean {
     return has(this.blockValidationStatuses, `block/${uuid}`)
   }
 
   hasResourceValidationErrors(uuidOrUuids: string | string[]): boolean {
     const uuids = castArray(uuidOrUuids)
 
-    for (let i = 0; i < uuids.length; i += 1) {
-      if (has(this.resourceValidationStatusesForActiveFlow, `resource/${uuids[i]}`)) {
-        return true
-      }
-    }
-    return false
+    return uuids.some(uuid => {
+      const isValid = this.validationStatuses[`resource/${uuid}`]?.isValid
+      const doesBelongToActiveFlow = this.activeFlow?.resources.some(resource => resource.uuid === uuid)
+
+      return isValid === false && doesBelongToActiveFlow
```

Comment:
`isValid` can be not only boolean, so a strict comparison with `false` is required

Reply:
Thanks for this, I like the usage of `some()`, which make an early return:
> As soon as one `true` is returned, `.some()` will itself return `true`